hdfscm/utils.py

```diff
@@ -71,5 +71,5 @@ def perm_to_403(path):
     except ArrowIOError as exc:
         # For now we can't access the errno attribute of the error directly,
         # detect it from the string instead.
```

Comment:
I can't find an issue for this but it seems like this has been fixed since this message was left 6 years ago, so we should be able to use:
```
import errno

if exc.errno == errno.EACCES:
``

Reply:
The errno approach worked! Thanks @jessie-ross!